doc/CONFIGURATION.md

```diff
@@ -859,72 +857,6 @@ const node = await createLibp2p({
 })
 ```
 
-#### Customizing Transports
```

Comment:
Should we replace this section with something other than webrtc-star?

Reply:
I agree we could replace it with webrtc and it's [`DataChannelOpts`](https://github.com/libp2p/js-libp2p-webrtc/blob/bc3ae14b9ff2b78fcd35616bcd7bf76c45b7a560/src/stream.ts#L13-L17) or [Websockets and its Multiaddr filter](https://github.com/libp2p/js-libp2p-websockets/blob/3645c2ba72ff1fd0b2a3f385f3cefa5e182f27ff/src/index.ts#L21)